hc/accounts/tests/test_profile.py

```diff
@@ -28,23 +33,33 @@ def test_it_sends_report(self):
         self.alice.profile.send_report()
 
         ###Assert that the email was sent and check email content
+        assert len(mail.outbox) > 0
+
+        self.assertIn("alice@example.org", mail.outbox[0].to)
+        self.assertIn("Monthly Report", mail.outbox[0].subject)
+        self.assertIn("This is a monthly report sent by ", mail.outbox[0].body)
 
     def test_it_adds_team_member(self):
         self.client.login(username="alice@example.org", password="password")
 
         form = {"invite_team_member": "1", "email": "frank@example.org"}
```

Comment:
In the spirit of proper variable naming, `form_data` or simply `data` might be a more apt name to use here, given that forms have a specific meaning in Django